src/common-ui/Toggle.tsx

```diff
@@ -0,0 +1,120 @@
+import { theme } from '@/styles/theme';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import Text from './Text';
+
+interface Props {
+  firstText: string;
+  secondText: string;
```

Comment:
```suggestion
  offText: string;
  onText  string;
```
또는 falseText, trueText 와 같이 각 text 가 의미하는걸 변수명으로 나타내주면 어떨까요??

Reply:
이게 훨씬 직관적이고 쓰는데 어려움이 없을 것 같네요 👍